scripts/utils.py

```diff
@@ -59,3 +62,66 @@ def is_old_format(uiConfig):
     if isinstance(uiConfig, dict):
         return False
     return True
+
+def parse_response(resp):
+    if resp.status_code >= 200 and resp.status_code <= 300:
+        return resp.status_code, resp.json()
+    else:
+        return resp.status_code, str(resp.content)
+
+def get_config_definition(base_url, selector, name="", auth=None):
+    request_url = f"{base_url}/{selector}-definitions/{name}"
+    if selector == "accounts":
+        request_url = f"{base_url}/account-definitions"
```

Comment:
Is this the right url ? we need API to fetch by name - @utsabc If webapp doesn't need it, we should have API for backend use-case

Reply:
I see we are loading all account definitions from DB at once for comparison - which is efficient way to do